mongo/integration/unified/event_verification.go

```diff
@@ -114,16 +120,26 @@ func verifyEvents(ctx context.Context, expectedEvents *expectedEvents) error {
 		return err
 	}
 
+	var ignoreExtraEvents bool
+	if expectedEvents.IgnoreExtraEvents != nil {
+		ignoreExtraEvents = *expectedEvents.IgnoreExtraEvents
+	}
```

Comment:
```suggestion
	ignoreExtraEvents := expectedEvents.IgnoreExtraEvents != nil && *expectedEvents.IgnoreExtraEvents
```
This block can be simplified to a single line.

Reply:
Sounds good; moved this logic to `verifyCommandEvents` and `verifyCMAPEvents`.